### PR TITLE
fix ES index name for todo-list example

### PR DESCRIPTION
**Pull request by guoyongzhang**
https://github.com/verrazzano/verrazzano/pull/555 changed how ES index name for OAM applications.

### CODE REVIEW

File: tests/e2e/examples/todo-list/todo_list_test.go [hunk 1]
```diff
@@ -198,22 +198,24 @@ var _ = ginkgo.Describe("Verify ToDo List example application.", func() {
 	//})
 
 	ginkgo.Context("Logging.", func() {
+		indexName := "oam-todo-list-todo-appconf-todo-domain"
```
Comment (markxnelson) [inline tests/e2e/examples/todo-list/todo_list_test.go:201]: is there a limit on how long these names can be?

**Review by markxnelson — APPROVED** [review]

File: tests/e2e/examples/todo-list/todo_list_test.go (hunk 1 above)
Comment (guoyongzhang) [reply tests/e2e/examples/todo-list/todo_list_test.go:201]: 255 characters

**Review by kminder — APPROVED** [review]

File: tests/e2e/examples/todo-list/todo_list_test.go [hunk 2]
```diff
@@ -198,7 +198,7 @@ var _ = ginkgo.Describe("Verify ToDo List example application.", func() {
 	//})
 
 	ginkgo.Context("Logging.", func() {
-		indexName := "oam-todo-list-todo-appconf-todo-domain"
+		indexName := "oam-todo-list--"
```
Comment (markxnelson) [inline tests/e2e/examples/todo-list/todo_list_test.go:201]: why two hyphens?

**Review by markxnelson — APPROVED** [review]

File: tests/e2e/examples/todo-list/todo_list_test.go (hunk 2 above)
Comment (guoyongzhang) [reply tests/e2e/examples/todo-list/todo_list_test.go:201]: It is due to a WLS fluentd bug known by Fred/John.  @fredtibbitts will fix it when he starts to work on WLS domain verrazzano wrapper.

**Review by kminder — APPROVED** [review]
Reapproving

**Pull request merged**